xm-commons-permission/src/main/java/com/icthh/xm/commons/permission/repository/PermittedRepository.java

```diff
@@ -141,10 +143,12 @@
         selectSql += WHERE_SQL + whereCondition;
         countSql += WHERE_SQL + whereCondition;
 
-        String permittedCondition = createPermissionCondition(privilegeKey);
-        if (StringUtils.isNotBlank(permittedCondition)) {
-            selectSql += AND_SQL + "(" + permittedCondition + ")";
-            countSql += AND_SQL + "(" + permittedCondition + ")";
+        Collection<String> permittedCondition = createPermissionCondition(privilegeKey);
+
+        if (isNotEmpty(permittedCondition)) {
+            String orChainedPermittedCondition = String.join(" OR ", permittedCondition);
```

Comment:
wrap each part to () for keep semantic
a mean that after join we must have () OR () instead of OR

Reply:
added